profiler/README.md

```diff
@@ -10,3 +10,4 @@ It should be considered experimental and is completely unsupported.
 |`splunk.profiler.enabled`            | false   | set to true to enable the profiler   |
 |`splunk.profiler.directory`          | "."     | location of jfr files                |
 |`splunk.profiler.recording.duration` | 20      | number of seconds per recording unit |
+|`splunk.profiler.keepfiles`          | false   | leave JFR files on disk id `true`    |
```

Comment:
Nit: why not `keep-files`?

Reply:
I dunno, I feel like the hyphen introduces a small parsing risk, but I'm not too paranoid about it.

Reply:
Not really, the upstream agent fully supports this: https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1414#issuecomment-733181398
We've basically copied the property naming scheme from spring

Reply:
Ok, I can change it.  👍🏻